tests/parametric/test_crashtracking.py

```diff
@@ -12,7 +12,7 @@
 @scenarios.parametric
 @features.crashtracking
 class Test_Crashtracking:
-    @irrelevant(context.library == "ruby@2.7.2-dev", reason="temporary skip for ruby 2.7.1 release")
+    @irrelevant(context.library == "ruby", reason="Test is failing")
```

Comment:
Could you add it as bug with a JIRA ticket ? 

```
    @bug(context.library == "ruby", reason="PROJECT-001")
```

Reply:
I suppose I can if I have permissions, which Jira project are you suggesting to file the ticket under?

Reply:
@p-datadog it "doesn't matter". we do put a bunch under `APMLP`. `PROF` could make sense.

Reply:
```suggestion
    @bug(context.library >= "ruby@2.7.2-dev", reason="APMLP-335")
```